From 3f72d36703d0bc43c2c4fc2f4780d3aa0143bfc4 Mon Sep 17 00:00:00 2001
From: WangJing <wangjing@mprc.pku.edu.cn>
Date: Mon, 10 Oct 2011 17:14:00 +0800
Subject: [PATCH 009/641] UniCore64 : Add definition of elf_fpregset_t

    Fixed error:
    include/linux/elfcore.h:135:40: error: expected declaration specifiers or '...' before 'elf_fpregset_t'
    include/linux/elfcore.h:137:90: error: expected declaration specifiers or '...' before 'elf_fpregset_t'
    include/linux/elfcore.h: In function 'elf_core_copy_task_fpregs':
    include/linux/elfcore.h:142:24: error: 'fpu' undeclared (first use in this function)
    include/linux/elfcore.h:142:24: note: each undeclared identifier is reported only once for each function it ap
    include/linux/elfcore.h:142:2: error: too many arguments to function 'dump_fpu'
    include/linux/elfcore.h:135:12: note: declared here

Signed-off-by: WangJing <wangjing@mprc.pku.edu.cn>
---
 arch/unicore64/include/asm/elf.h |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/unicore64/include/asm/elf.h b/arch/unicore64/include/asm/elf.h
index bb7a783..3847867 100644
--- a/arch/unicore64/include/asm/elf.h
+++ b/arch/unicore64/include/asm/elf.h
@@ -15,4 +15,6 @@ typedef elf_greg_t elf_gregset_t[ELF_NGREG];
 #define ELF_PLATFORM		(NULL)
 #define ELF_EXEC_PAGESIZE	PAGE_SIZE
 
+typedef struct fpu_regs elf_fpregset_t;
+
 #endif /* __ASM_UNICORE64_ELF_H__ */
-- 
1.7.9.5

